window-manipulate/window.js

```diff
@@ -0,0 +1,60 @@
+document.addEventListener("click", function(e) {
+
+  function callOnCurrentWindow(callback){
+    chrome.windows.getCurrent(
+      function(currentWindow) {
```

Comment:
Since you're already using arrow functions elsewhere, I'd suggest using them everywhere because they make the code more concise.